README.md

```diff
@@ -1,4 +1,47 @@
 metro
 =====
 
-Module for sending metric to configurable pipelines
+Node module for sending metrics to graphite time series DB
+
+How to use metro in your repo:
+==============================
+
+1. `npm install metro -s`
+
+2. As part of the setup process create factory for graphite pipeline. Recommended to do it in a dedicated file. Example:
+
+    const config = require('config');
+    const metro = require('metro');
+
+    module.exports = function setupMetro() {
+
+      let metroClient = null;
+      metro.create('default');
+      if (config.graphite.host) {
+        metroClient = metro.get('default');
+        metroClient.addPipeline(metro.pipelineCreator('graphite', config.graphite.prefix, {
+          host: config.graphite.host,
+          port: config.graphite.port
+        }));
+      }
+
+      return metroClient;
+    };
+
+3. init metro in your app.js, example:
+
+    const metro = require('setup/metro');
+
+    const app = bootleg();
+    app.phase('metro', metro, config);
+
+4. send data, example:
+
+    const metro = require('metro');
+    const metroSender = metro.get('latency_route');
+    if (metroSender !== null) {
+
+        metroSender.send(`latencies.${organizationName}.sentinel.authorization`,
```

Comment:
I'd remove our key with a sample key, E.g: `'latencies.appname.response_time` or something simple like that, ditto with the value, just a simple number to show how it's used